website/static/js/filepage/index.js

```diff
@@ -225,41 +221,41 @@ var FileViewPage = {
             m('.btn-group.m-t-xs', [
                 m('.btn.btn-sm.btn-primary.file-download', {onclick: $(document).trigger.bind($(document), 'fileviewpage:download')}, 'Download')
             ]),
-            m('.btn-group.btn-group-sm.m-t-xs', [
-                m('.btn.btn-default.disabled', 'Toggle view: ')
+            m('.btn-group.btn-group-sm.m-t-xs' + (ctrl.editor ? '.btn-group' : ''), [
```

Comment:
If I remember correctly, the reasoning for this was either that otherwise the 'View' button ended up not being the right size or the left edge remained not rounded. Will play around with it to make sure it's needed.